main.py

```diff
@@ -115,6 +115,14 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: string. The route to be redirected.
+        handler: callable or string. Custom handler method to be called.
+        defaults: dict or default. Default or extra keywords to be returned by this route.
```

Comment:
are you sure about the type? I think this can be `dict | None` but not sure, can you please check?

Reply:
Thanks , I have also changed the **dict or default** to **dict | None** . I had used default to mean the current default value which was None but also realised that wasn't descriptive enough. I also checked , and saw in the docs that it can be dict or None.